Makefile

```diff
@@ -1,12 +1,12 @@
-install-test:
-	mkdir -p vendor/sass
-	cd vendor/sass && git clone https://github.com/sass/sass-spec.git && cd ../..
-
-test:
+test: vendor
 	vendor/bin/phpunit --colors tests
 
-sass-spec:
+sass-spec: vendor
 	TEST_SASS_SPEC=1 vendor/bin/phpunit --colors tests 2>&1 | tee /tmp/sass-spec.log | tail -2
 
-standard:
+standard: vendor
 	vendor/bin/phpcs -s --standard=PSR12 --exclude=PSR12.Properties.ConstantVisibility --extensions=php bin src tests *.php
+
+vendor: composer.json
+	composer update
+	touch $@
```

Comment:
this `touch vendor` ensures that the modified date of the `vendor` folder itself is updated. This way, `make` will detect that `vendor` is uptodate when you run `make test` and won't run the command again (nice as you will run tests much more often than changing the `composer.json` when working on features)